internal/telemetry/metrics.go

```diff
@@ -0,0 +1,162 @@
+package telemetry
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"runtime"
+	"time"
+
+	"github.com/jzelinskie/cobrautil"
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
+	"github.com/rs/zerolog/log"
+	"golang.org/x/sync/errgroup"
+
+	"github.com/authzed/spicedb/internal/datastore"
+	"github.com/authzed/spicedb/internal/middleware/usagemetrics"
+)
+
+var (
+	Registry                  = prometheus.NewRegistry()
```

Comment:
I don't think we need to export this

Reply:
I use it to export the telemetry with a different handler on `/telemetry`

Reply:
Added a method to just create a handler. Having the registry exported could invite abuse and attempts at data exfiltration.